app/components/global/AppBar/ProfileMenu.js

```diff
@@ -144,7 +144,9 @@ class ProfileMenu extends React.Component {
             <PopOverPanel>
               <Menu>
                 <MenuItem>
-                  <MenuHeading>{user.identities[0].name}</MenuHeading>
+                  <MenuHeading data-hj-suppress="">
```

Comment:
May be worth adding a test for this. It'll help us understand that it's for hotjar.

Reply:
Agreed.... if its more than 20mins then use another PR :+1:

Reply:
Not sure what do you think it can be tested.
As far  as I know the issue is, hotjar is recording customer navigation, so what is required is in those recording to suppress certain fields, this behaviour cannot be reproduced in our website rendering the page, the only thing can be saw is that the property `data-hj-suppress` is present in the dom object, as you can see in the attached screenshot.

To check if this is actually suppressing the field, I'm afraid the only way is going to the hotjar website and checking whether the recordings don't contain the suppressed fields.

![screenshot 2019-02-22 at 09 40 30](https://user-images.githubusercontent.com/5822727/53234115-bdd6cf80-3686-11e9-97ad-c083202cd970.png)

Reply:
I think if we assert that the menu heading containing the name of the user has that attribute, it allows us to write a test explaining it's for suppressing the value from hotjar, i.e. `it('should be suppressed from hotjar')`

We can assume the library is tested already and will respect the attribute.